src/test/java/org/jenkinsci/plugins/workflow/libs/LibraryAdderTest.java

```diff
@@ -290,7 +292,7 @@ public static void register(Object o) {
             clearInvocationCaches.invoke(metaClass);
         }
         for (WeakReference<ClassLoader> loaderRef : LOADERS) {
-            MemoryAssert.assertGC(loaderRef);
+            MemoryAssert.assertGC(loaderRef, false);
```

Comment:
Picked up from `workflow-cps` and verifies that there are no soft, much less hard, leaks.